src/main/java/app/entities/Game.java

```diff
@@ -0,0 +1,62 @@
+package app.entities;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDate;
+import java.util.Set;
+
+@Entity
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@NamedQueries({
+        @NamedQuery(name = "Game.getAll", query = "SELECT g FROM Game g")
+})
+@Table(name = "game")
+public class Game {
+
+    @Id
+    @Basic(optional = false)
+    @Column(unique = true, nullable = false)
+    private Long id;
+
+    @Basic(optional = false)
+    private String title;
+
+    @Basic(optional = false)
+    @Column(name = "release_date")
+    private LocalDate releaseDate;
+
+    @Column(name = "background_image_url")
+    private String backgroundImageURL;
+
+    @Column(name = "meta_critic_score")
+    private Double metaCriticScore;
```

Comment:
The metaCriticScore should be an Integer as it is from the ([API documentation](https://api.rawg.io/docs/#operation/games_read)). (Also kindly change the GameDTO to use Integer instead of Double)